guidelines/backend-coding-standard/backend-coding-standard.md

```diff
@@ -0,0 +1,40 @@
+# Backend Coding Standard Draft
+
+## Coding style
+- function should be stateless, pure
+- function should not have side effect, i.e. interact with the environment outside the function, causing mutatation with other things that is not expected to change
+- input id instead of the object itself to the function, then query by id inside the function
```

Comment:
I would say this point is not only ID vs object. this is more related to how we want to reduce the coupling.

Maybe move this point to module section first since this is more related to that context